src/Sylius/Component/Addressing/Model/AddressLogEntry.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Addressing\Model;
+
+use Sylius\Bundle\ResourceBundle\Model\ResourceLogEntry;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+final class AddressLogEntry extends ResourceLogEntry
```

Comment:
🐃  Maybe we should point `ResourceLogEntry` as an address log entry in configuration, cuz this class has no sense to me it is empty and final 🌵.